includes/header.php

```diff
@@ -1,3 +1,5 @@
+<?php require 'app/functions.php'; ?>
+
```

Comment:
Yeah I've just seen that too. Shouldn't be there... not sure why that didn't update when I redid the commit